manifests/mc2.pp

```diff
@@ -0,0 +1,19 @@
+# == Class: seed_stack::mc2
+#
+# Runs a Mission Control container in Marathon.
+#
+class seed_stack::mc2(
+  $infr_domain,
+  $hub_domain,
+  $marathon_host = 'marathon.service.consul',
+) {
+  file { '/etc/marathon-apps': ensure => 'directory' }
+  ->
+  file { '/etc/marathon-apps/mc2.marathon.json':
```

Comment:
I don't think xylem should mandate the location of these files. They might live on a shared volume or something.